docs/logs-events/log-restoration/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Restore archived logs
+shorttitle: Restore logs
+weight: 600
+---
+
+The log restoration feature of One Eye allows you to retrieve a set of logs into an Elasticsearch instance where you can browse and analyze them. This is useful for example in auditing and troubleshooting situations.
+
+## Prerequisites
+
+- Restoring logs is supported only from S3 output
+- The elastic operator must be installed on the cluster running One Eye
+- The cluster running One Eye must have enough resources to store the restored logs, and deploy an Elastic instance. The exact requirements depend on the amount of logs to be restored.
+
+## Restore logs
+
+To restore archived log messages, complete the following steps.
+
+1. [Open the One Eye web interface]({{< relref "/docs/one-eye/quickstart/_index.md#display-the-logging-flow" >}}).
+1. Select **MENU > LOG RESTORATION**, then click ![Create new icon](/docs/one-eye/headless/icon-create-new.png).
+    ![Restored logs list](restored-logs-list.png)
+1. Click ![Create new icon](/docs/one-eye/headless/icon-create-new.png). The Log Restoration Wizard opens.
+    ![Log Restoration Wizard](log-restoration-output.png)
+1. Select the output from which you want to restore logs, then click **Next**.
+1. Select the cluster and the time interval for which you want to restore logs.
+    ![Log Restoration Wizard](log-restoration-cluster.png)
+1. To specify the details of the logs to be restored on the UI, select **Simple Builder**. Alternatively, you can select **Advanced Builder**, and paste a [JSON that describes the logs to restore](#builder-json).
+1. To restore the logs of only specific namespaces, pod, and containers, search for the name of the respective resource, and select the ones you need. If you want to restore the logs of every resource, select **Any**. Note that searching for the resources returns only exact matches, unless you include the * wildcard.
+    ![Filter logs to restore](log-restoration-namespace.png)
+1. Set a prefix for the Elastic index. In addition to using it as a prefix for the Elastic index, the Elastic resources are named after this parameter.
+    ![Set Elastic parameters](log-restoration-configuration.png)
+1. Set the namespace where the log restoration job will run. The Elastic instance is also deployed into this namespace.
+1. Before restoring the logs starts, review the settings of the process. If everything is as you intended, click **Next**.
+    ![Log Restoration overview](log-restoration-overview.png)
+1. One Eye starts restoring the logs. Depending on the amount of logs to restore, this can take a long time.
+    ![Log Restoration overview](log-restoration-progress.png)
+1. After the job is successfully completed, you can [access the restored logs](#access-logs).
+
+## Access restored logs {#access-logs}
+
+To tail your logs, or the logs that match a specific query in real-time, complete the following steps.
```

Comment:
I either misunderstand what you're trying to say or it's just wrong. These are not real-time logs.
```suggestion
To view your restored logs, complete the following steps.
```